client/components/UI/SlideInPanel/Content.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Content = ({children}) => (
+    <div className="sd-slide-in-panel__content">
+        <div className="sd-slide-in-panel__content-block sd-slide-in-panel__content-block--first">
```

Comment:
BEM modifiers should probably be a property of the Component.
i.e. `sd-slide-in-panel__content-block--first` should be controlled via a `first` boolean property